### PR TITLE
community/caddy: upgrade to 0.11.0

### PR DESCRIPTION
**Pull request by xpecex**
ref https://caddyserver.com/blog/caddy-0_11-released

### CODE REVIEW

**Comment by andypost** [comment]
It looks better then https://github.com/alpinelinux/aports/pull/5155

File: community/caddy/APKBUILD [hunk 1]
```diff
@@ -53,15 +54,22 @@ package() {
...
-	install -Dm755 "$srcdir"/$pkgname.initd \
-		"$pkgdir"/etc/init.d/$pkgname
 	install -Dm644 "$srcdir"/$pkgname.confd \
 		"$pkgdir"/etc/conf.d/$pkgname
 	install -Dm644 "$srcdir"/$pkgname.conf \
 		"$pkgdir"/etc/$pkgname/$pkgname.conf
 }
 
-sha512sums="b2d7ff08687c1b1331c018430e3458d4a5e4d5923803fabe563536ce372209ac0350efd4112a1099bfbfb517935d4f498613820d6b0685d5947a903227e15636  caddy-0.10.14.tar.gz
+_openrc() {
```
Comment (nmeum) [inline community/caddy/APKBUILD]: There should be no need for this, just install the services from `package()` and use the [default `-openrc` subpackage function](https://git.alpinelinux.org/cgit/abuild/tree/abuild.in#n1723).

Comment (xpecex) [reply community/caddy/APKBUILD]: fixed, thanks.

**Comment by andypost** [comment]
It could be commited

File: community/caddy/APKBUILD [hunk 2]
```diff
@@ -53,15 +54,13 @@ package() {
 		"$pkgdir"/etc/caddy \
 		"$pkgdir"/var/www
 
-	install -Dm755 "$srcdir"/$pkgname.initd \
-		"$pkgdir"/etc/init.d/$pkgname
```
Comment (clandmeter) [inline community/caddy/APKBUILD]: Why are installation of init.d removed here?

Comment (andypost) [reply community/caddy/APKBUILD]: I guess because they provide own https://github.com/mholt/caddy/tree/master/dist/init

Comment (clandmeter) [reply community/caddy/APKBUILD]: I don't see any openrc related files in that location.

**Comment by andypost** [comment]
Already upgraded to 0.11.1 https://github.com/alpinelinux/aports/commit/1882ba6820b0cdd50de5aae9c68fa1c1ce75ae2c

**Comment by nmeum** [comment]
> Already upgraded to 0.11.1 1882ba6

Ok, closing then.

**Pull request closed without merging**